README.md

```diff
@@ -34,15 +34,24 @@ After compiling a Cairo0 program to `program.json`, run:
 stone-prover-cli prove program.json
 ```
 
-### Run and prove one or more programs/PIEs with the Starknet bootloader
+### Run and prove one or more programs/PIEs
+
+#### Running with the Starknet bootloader
 
 If you want to prove one or more programs and PIEs by running them with the Starknet bootloader,
-you can use the `--with-bootloader` option.
+you can use the `--with-bootloader` option which is enabled by default.
```

Comment:
The CLI uses the Cairo bootloader by default to prove one or more programs and PIEs.